ExampleFastfile

```diff
@@ -1,4 +1,4 @@
-fastlane_version "1.55.0"
+fastlane_version "2.51.0"
```

Comment:
#? why downgrade?

Reply:
how do you mean downgrade? I'm upgrading from 1.55 to 2.51

Reply:
Oh hey, that's a 2, not a 1. I blame monospace fonts and non-Retina displays. 😳

Reply:
haha no worries